### DISCUSSION TITLE
Error when trying expand

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/keto#576] [Q&A] [answered]
Hey!

I created some relation tuples by using the REST API. 
I also checked a relation via https://www.ory.sh/keto/docs/reference/api#check-a-relation-tuple. However the expand query is not working for me.

When doing
`
curl -H "Content-Type: application/GET http://127.0.0.1:4466/expand?namespace=team&object=productowner&relation=member`

i get
`
{"error":{"code":400,"status":"Bad Request","message":"strconv.ParseInt: parsing \"\": invalid syntax"}}`

Any ideas what i am doing wrong?

**Comment by zepatrik** [comment, marked as answer]
I am pretty sure that expand requires the `max-depth` parameter, which is wrongly not marked as required in the API documentation: https://www.ory.sh/keto/docs/reference/api/#parameters-1

**Reply by zepatrik** [reply]
Also that message should include the parameter name to make debugging easier. Would you be up for the challenge? The changes would be required here: https://github.com/ory/keto/blob/1e047468b14e81969beb5edd7cb4561edfacb88f/internal/expand/handler.go#L78

**Reply by pommelinho** [reply]
hey thanks for your  reply.

but also when i do 

` curl -H "Content-Type: application/GET" http://127.0.0.1:4466/expand?namespace=team&object=productowner&relation=member&max-depth=1` 
the same error is thrown

**Reply by zepatrik** [reply]
It might be related to some bash escape problems then maybe? `&` means that the command should run in the background. Either try to escape/quote or just use
```
curl -G --silent \
     --data-urlencode "namespace=files" \
     --data-urlencode "relation=access" \
     --data-urlencode "object=/photos/beach.jpg" \
     --data-urlencode "max-depth=3" \
     http://127.0.0.1:4466/expand
```
as in https://www.ory.sh/keto/docs/guides/expand-api-display-who-has-access

The docs code samples are tested in the CI so I am pretty sure it works :wink:

**Reply by pommelinho** [reply]
Thanks worked! ;-)

`curl -H "Content-Type: application/GET" 'http://127.0.0.1:4466/expand?namespace=team&object=productowner&relation=member&max-depth=3'`
